frontend/src/app/trees/forests/tree-guidelines/tree-cutting/tree-tools/tree-tools.component.html

```diff
@@ -21,8 +21,8 @@
       <div class="usa-width-one-third tree-image-container">
         <img src="/assets/img/no-snowmobile.jpg" alt="snowmobiles are prohibited" />
         <img src="/assets/img/cutting-tools.gif" alt="hacksaw, rope, tarp, gloves" />
-        <span class="caption">Tools you will need</span>
+        <figcaption class="caption">Tools you will need</figcaption>
```

Comment:
[Per W3C](https://www.w3.org/wiki/HTML/Elements/figcaption), `figcaption` needs to be a child of the `figure` element. `figure` can have multiple children, so the solution here is to wrap both `img` tags and the `figcaption` in `<figure>`...`</figure>`.